build_tools/fetch_sources.py

```diff
@@ -88,6 +91,31 @@ def run(args):
         apply_patches(args, projects)
 
 
+def pull_large_files(dvc_projects, projects):
+    if not dvc_projects:
+        print("No DVC projects specified, skipping large file pull.")
+        return
+    if shutil.which("dvc") is None:
+        print("Could not find dvc on PATH so large files could not be fetched")
+        print("To install dvc, run pip install dvc or pip install -r requirements.txt")
```

Comment:
I'd prefer if the keywords/phrases here were highlighted using some style of quotes:
```
`dvc`
'dvc'
"dvc"
```

```suggestion
    if shutil.which("dvc") is None:
        print("Could not find `dvc` on PATH so large files could not be fetched")
        print("To install dvc, run `pip install dvc` or `pip install -r requirements.txt`")
```